pkg/kubectl/cmd/certificates.go

```diff
@@ -225,9 +226,8 @@ func (options *CertificateOptions) modifyCertificateCondition(builder *resource.
 			csr := info.Object.(*certificatesv1beta1.CertificateSigningRequest)
 			csr, hasCondition := modify(csr)
 			if !hasCondition || force {
-				csr, err = clientSet.Certificates().
-					CertificateSigningRequests().
-					UpdateApproval(csr)
+				var client certificatesv1beta1client.CertificatesV1beta1Interface
```

Comment:
You need to pass the client here as before, and most importantly you need to create it.